boards/BMS/Inc/Io/Io_LTC6813.h

```diff
@@ -3,6 +3,17 @@
 #include <stm32f3xx_hal.h>
 #include "App_SharedExitCode.h"
 
+enum CellVoltageRegisterGroups
+{
+    CELL_VOLTAGE_REGISTER_GROUP_A,
+    CELL_VOLTAGE_REGISTER_GROUP_B,
+    CELL_VOLTAGE_REGISTER_GROUP_C,
+    CELL_VOLTAGE_REGISTER_GROUP_D,
+    CELL_VOLTAGE_REGISTER_GROUP_E,
+    CELL_VOLTAGE_REGISTER_GROUP_F,
+    NUM_OF_CELL_VOLTAGE_REGISTER_GROUPS
+};
```

Comment:
Can we put this in the `.c` file? Generally we want to minimize the amount of stuff in the header to make it as obvious and simple to the user what they're supposed to call/use.

Reply:
Also should this be called `CellVoltageRegisterGroup` (ie. not plural) since each entry in it is a single group?

Reply:
Okay this is moved to the .c file, and the enum's name is also fixed